src/components/grid-aware/BlockQuoteBlock/BlockQuoteBlock.module.css

```diff
@@ -0,0 +1,63 @@
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    grid-template-columns: 30px 1fr 30px;
```

Comment:
Is it worth setting up a grid here? You don't actually use the 30px margins from here, since you separately set that up in the padding elsewhere.

You might just want to set `display: block` to get rid of the `grid`. If you do that, then you don't need to set `grid-template-columns` in the media query, since it will no longer behave like a grid.

Reply:
👍 Oh, changed grid-template-columns to display.